saveEventsFile.m

```diff
@@ -1,4 +1,4 @@
-function [logFile] = saveEventsFile(action, expParameters, logFile, varargin)
+function [logFile] = saveEventsFile(action, expParameters, logFile)
```

Comment:
I'm missing it somewhere probably, now does this function not throw an error if "save" option is chosen but the logfile is empty?

Reply:
Good catch ! :rocket: 

Will add a few more input checks and some additional errors so that users know what is expected of them.

Reply:
This should be fixed now.

If logFile is empty it is initialized but before running save it checks that the fileID field exists and is not empty.

It will throw an error in case of problem.